src/CampaignKit.WorldMap/Services/DefaultRandomDataService.cs

```diff
@@ -41,17 +28,31 @@ public interface IRandomDataService
     /// <seealso cref="T:CampaignKit.WorldMap.Services.IRandomDataService" />
     public class DefaultRandomDataService : IRandomDataService
     {
-        #region Fields
-
-        private readonly Random _rand = new Random();
-
-        #endregion
+        /// <summary>
+        /// The pseudo-random number generator.
+        /// </summary>
+        private readonly Random rand = new Random();
 
-        #region Implementations
+        /// <summary>
+        /// The application configuration.
+        /// </summary>
+        private readonly IConfiguration configuration;
 
-        #region IRandomDataService Members
+        /// <summary>
+        /// The application logging service.
+        /// </summary>
+        private readonly ILogger loggerService;
 
-        #region Public Methods
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DefaultRandomDataService"/> class.
+        /// </summary>
+        /// <param name="configuration">The application configuration.</param>
+        /// <param name="loggerService">The logger service.</param>
+        public DefaultRandomDataService(IConfiguration configuration, ILogger<DefaultRandomDataService> loggerService)
+        {
+            this.configuration = configuration ?? throw new ArgumentNullException(nameof(configuration));
```

Comment:
Are these two services needed for anything?

Reply:
No.  I'll delete them.